### ISSUE TITLE
Error in creating Personas

### ISSUE THREAD

**Issue by ManuXD32** [Vali-98/ChatterUI#16]
Hey, first of all I wanted to thank you for all your hard work, this app is amazing.

I'm using the last version (v 0.7.3) an I have noticed that I cannot create any Personas, I hit the + button, write my name but nothing happens. I also can't edit the description of the default persona or change the Persona's picture nor the character's ones.

**Comment by GameOverFlowChart** [comment]
I think that feature isn't implemented yet. 

Also here is a related log:
```
[23:56:15] : Could not generate default User. Reason: Error: Call to function 'ExponentFileSystem.readDirectoryAsync' has been rejected.
→ Caused by: Uri 'file:///data/user/0/com.Vali98.ChatterUI/files/persona' doesn't exist or isn't a directory
[23:56:15] : Resetting state values for startup.
```

**Comment by Vali-98** [comment]
This seems to be an issue for new installations of the app, as I accidentally removed the generation of the default personas directory due to an incomplete transition to SQLite.

I will likely prioritize completing the SQL migration of personas which should resolve this issue.

**Comment by Vali-98** [comment]
This should be fixed in f96497c98646e16df46af86b34a9929ddcaac334

**Comment by ManuXD32** [comment]
Fixed. Just tried it and it works like a charm. Thanks!!!!
